### PR TITLE
Adding podspec for GTMSessionFetcher.

### PR DESCRIPTION
**Pull request by jayneflux**

### CODE REVIEW

File: GTMSessionFetcher.podspec [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  as a wrapper on NSURLSession, so its behavior is asynchronous 
+  and uses operating-system settings on iOS and Mac OS X.
+  DESC
+  
+  s.ios.deployment_target = '7.0'
+  s.osx.deployment_target = '10.8'
+
+  s.compiler_flags = '-DGTM_USE_SESSION_FETCHER=1'
+
+  s.default_subspec = 'Core', 'Full'
```
Comment (perotinus) [inline GTMSessionFetcher.podspec]: Should this be 'default_subspecs'? It looks like the syntax pluralizes it when there is more than one.

**Comment by jayneflux** [comment]
Default has been left at Full since it includes Core anyway.

File: GTMSessionFetcher.podspec [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+# to import GTMSessionFetcher via the CocoaPods dependency Manager.
+Pod::Spec.new do |s|
+  s.name        = 'GTMSessionFetcher'
+  s.version     = '1.1.0'
+  s.authors     = 'Google Inc.'
+  s.license     = { :type => 'Apache', :file => 'LICENSE' }
+  s.homepage    = 'https://github.com/google/gtm-session-fetcher'
+  s.source      = { :git => 'https://github.com/google/gtm-session-fetcher.git',
+                    :tag => "v#{s.version}" }
+  s.summary     = 'Google Toolbox for Mac - Session Fetcher.'
```
Comment (perotinus) [inline GTMSessionFetcher.podspec]: nit: can you remove the trailing period?

**Comment by perotinus** [comment]
Looks like the travis failure was a flaky test, not anything that you've done. I think this is good to do (perhaps modulo the one nit comment I left.)

**Pull request merged**